### ISSUE TITLE
No quick add?

### ISSUE THREAD

**Issue by bucketdraws** [successible/cleanslate#121]
Hi! Sorry if this is something I'm just not seeing! 
I can't find how to add just some calories to my day, without having to create a food  or a recipe. 
In the tour video it's the little rocket button, but when I click on it it brings up "create food". 

Thanks for the app, btw, I just started using it and I love the concept and philosophy behind it!

**Comment by paulzakin** [comment]
thank you :)

I just logged in to the website to double check - it is working!

The magnifying glass will let you add a food via search and the rocket will let you add calories and protein directly. 

Maybe post a GIF or screenshot so I can get a better sense of what is happening?

**Comment by bucketdraws** [comment]
Hey! Thank you for the really fast reply!! 
I tried to quick add many times yesterday throughout the day and it didn't work either on my pc nor on my android phone. 

Last night I did this video before falling asleep to show the issue. This morning I tried to replicate it on my pc to send both examples, but it turns out it now works perfectly on both pc and android. So nevermind, I guess! Sorry to bother you. 
I have no idea what went wrong yesterday :')

https://github.com/successible/cleanslate/assets/171722858/c9f0bfe9-9bc8-4b8c-af52-84288f381739
